speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/ConfigEnums.h

```diff
@@ -3,7 +3,7 @@
 /// @brief The different tasks that can be performed
 typedef unsigned int Task;
 
-#define TASK_NO_TASK        0
+#define TASK_NO_TASK        -1
 #define TASK_LANE_KEEPING   1
 #define TASK_SPEED_CONTROL  2
```

Comment:
Now that no task is no longer 0, lane keeping and speed control can be 1 lower

Reply:
implementing this change brakes the code!

Reply:
I fixed this for you, make sure to delete your local config files in case it stored out-of-bounds settings.